pos_invoice_send_mail/static/src/js/screens.js

```diff
@@ -0,0 +1,60 @@
+/* Copyright 2019 Druidoo - Iván Todorovich
+   License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl). */
+   
+odoo.define('pos_invoice_send_mail.screens', function (require) {
+    "use strict";
+
+var core = require('web.core');
+var screens = require('point_of_sale.screens');
+var _t = core._t;
+
+screens.PaymentScreenWidget.include({
+    renderElement: function() {
+        var self = this;
+        this._super();
+        this.$('.js_send_mail').click(function(){
+            self.click_send_mail();
+        });
+    },
+    click_send_mail: function() {
+        var order = this.pos.get_order();
+        order.set_to_send_mail(!order.is_to_send_mail());
+        if (order.is_to_send_mail()) {
+            this.$('.js_send_mail').addClass('highlight');
+        } else {
+            this.$('.js_send_mail').removeClass('highlight');
+        }
+        if (order.is_to_send_mail() && !order.is_to_invoice()) {
+            this.click_invoice();
```

Comment:
Question : it's not better to hide the button (send by email) if invoice button is not checked ?

Reply:
Hmm, I thought about it but it's much faster for the cashier to do it like this